.github/workflows/build.yml

```diff
@@ -119,3 +119,12 @@ jobs:
           cd "$GITHUB_WORKSPACE"
           (cd ghci-wrapper && cabal v1-configure --enable-tests --ghc-options=-Werror $CABAL_OPTS && cabal v1-build && cabal v1-test)
           cabal v1-configure --enable-tests --ghc-options=-Werror $CABAL_OPTS && cabal v1-build && cabal v1-test
+
+  success:
```

Comment:
@andreasabel I added a branch protection rule for "main" that requires that the "success" job has to pass before merging.  This may look convoluted, however, I'm not aware of a simpler approach without listing all the jobs explicitly.  GitHub does not allow you to say "all jobs have to pass" before merging.

Reply:
In the `haskell/cabal` project, there is apparently an explicit list of workflows that have to pass.